spec/datadog/core/telemetry/collector_spec.rb

```diff
@@ -176,6 +176,48 @@
         it { is_expected.to_not include(:DD_TRACE_SAMPLE_RATE) }
       end
     end
+
+    context 'DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED' do
+      around do |example|
+        ClimateControl.modify(
+          DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED: dd_trace_remove_integration_service_names_enabled
+        ) do
+          example.run
+        end
+      end
+
+      let(:dd_trace_remove_integration_service_names_enabled) { nil }
```

Comment:
Minor: I don't think this is needed? (Maybe a left-over from a previous test?) (Same applies below)